website/docs/r/file_instance.html.markdown

```diff
@@ -0,0 +1,141 @@
+---
+# ----------------------------------------------------------------------------
+#
+#     ***     AUTO GENERATED CODE    ***    AUTO GENERATED CODE     ***
+#
+# ----------------------------------------------------------------------------
+#
+#     This file is automatically generated by Magic Modules and manual
+#     changes will be clobbered when the file is regenerated.
+#
+#     Please read more about how to change this file in
+#     .github/CONTRIBUTING.md.
+#
+# ----------------------------------------------------------------------------
+layout: "google"
+page_title: "Google: google_file_instance"
+sidebar_current: "docs-google-file-instance"
+description: |-
+  A Google Cloud Filestore instance.
+---
+
+# google\_file\_instance
```

Comment:
I kind of want this to be named `google_filestore_instance`. What do you think?

Reply:
I really do - what do we do to change the name away from the actual API name?  I couldn't figure that one out just from looking, and I didn't spend more than a few minutes poking around.

Reply:
I think it's the `prefix` field in api.yaml (or at least I hope that's all that field does).